### ISSUE TITLE
[BUG] Plot mover isn't working

### ISSUE THREAD

**Issue by tekstaker** [tydeno/Ploto#105]
**Describe the bug**
Plot mover isn't working for me.  After a while, the outdrive just fills up, and the ploto script terminates.

**To Reproduce**
I have a bunch of paths.  Some exist, some are full.

eg:
```
...
    "EnableMover":  "true",
    "PathsToMovePlotsTo":  "H:\\chia_nft_plots,I:\\chia_nft_plots,J:\\chia_nft_plots,L:\\chia_nft_plots,M:\\chia_nft_plots,O:\\chia_nft_plots,P:\\chia_nft_plots",
    "DiskConfig":  [
...

```
**Expected behavior**
- an error should be generated for missing folders.
- a warning should be generated for full folders.
- the script should try to fill the drives, starting with the first one in the list and working it's way to the last drive.
- it would be nice if the UI returned information about the status of the mover process.  eg: No plots found to move, sleeping for 60 seconds. ;  Moving plot from E:\\adfadfasdfdsf.plot to G:\\PlotArchive\adfadfasdfdsf.plot  @ 12.6MB/s, ETA 1.2hrs.
- it would be nice if the mover could move the plots in parallel - I have a SSD outdrive, but the final drives are spinners.  So I could actually move 4 plots from the SSD to 4 different drives at the same time, without the SSD being the bottleneck.

**Ploto Version in use**
 - it would be nice if ploto spawns showed the version number.
 - 
![image](https://user-images.githubusercontent.com/1213877/125203067-b374fa80-e26e-11eb-8818-4603750a378c.png)

**Additional context**
@killadreams says that maybe we should use a single forward slash instead of the double backslashes in the PathsToMovePlotsTo parameter.

**Comment by tekstaker** [comment]
Here's a screenshot of the plotmover NOT finding valid plots in the outdrive: 
![image](https://user-images.githubusercontent.com/1213877/125205458-436c7180-e27a-11eb-99e5-ea5bc13f9f9d.png)


As you can see, there are two files in the outdrive awaiting transfer.

**Comment by chefsas** [comment]
@lanceosaurus can you confirm if Ploto-Mover is copying the files to the final destination you have specified in your config? I figured out that's what Ploto was doing with mine. They were being copied to the final destination, but Ploto would not delete them from the staging drive (source). I ended up having to setup a robocopy script that monitors the folders periodically for .PLOT files and moves them. That seems to be working for me right now.

**Comment by tekstaker** [comment]
No, it's not doing anything.  The plots are not copied to the destination,
they are just ignored.

**Comment by chefsas** [comment]
FYI -- here's the script I leave running in PowerShell -- it scans when 2 changes are detected (/mon:2) and moves the plots from E:\ to Z:\ (which is a network share on my farmer). I have multiple staging drives, so I just open multiple PowerShell windows and adjust the drive paths as necessary. This is messy because it isn't smart enough to move to the next final drive when Z:\ fills like Ploto would....

robocopy /mov /mon:2 E:\ Z:\ *.PLOT

**Comment by tekstaker** [comment]
Yes, I too have a manual script to move them.... but they are supposed to be automagically moved by Ploto!
